build/install-tool.sh

```diff
@@ -1,10 +1,16 @@
 #!/bin/bash
 
-. /usr/local/build/util.sh
+# . /usr/local/build/util.sh
```

Comment:
```suggestion
 . /usr/local/build/util.sh
```

This is needed by all scripts, passed by this script, tool script will be sourced with my pr